utils/src/main/java/com/cloud/utils/ssh/SshHelper.java

```diff
@@ -196,6 +201,16 @@ public static void scpTo(String host, int port, String user, File pemKeyFile, St
 
             String result = sbResult.toString();
 
+            if (StringUtils.isBlank(result)) {
+                try {
+                    result = IOUtils.toString(stdout, UTF8);
```

Comment:
Rather than use a homegrown string constant, pass ``StandardCharsets.UTF_8`` instead.  An additional benefit is that it is a ``Charset`` instance which provides greater type safety.